openfpga_flow/scripts/run_fpga_flow.py

```diff
@@ -692,12 +694,27 @@ def extract_vpr_stats(logfile, r_filename="vpr_stat", parse_section="vpr"):
 
 def run_rewrite_verilog():
     # Rewrite the verilog after optimization
-    script_cmd = [
-        "read_blif %s" % args.top_module+".blif",
-        "write_verilog %s" % args.top_module+"_output_verilog.v"
-    ]
-    command = [cad_tools["yosys_path"], "-p", "; ".join(script_cmd)]
-    run_command("Yosys", "yosys_rewrite.log", command)
+    if not args.ys_rewrite_tmpl:
+        script_cmd = [
+            "read_blif %s" % args.top_module+".blif",
+            "write_verilog %s" % args.top_module+"_output_verilog.v"
+        ]
+        command = [cad_tools["yosys_path"], "-p", "; ".join(script_cmd)]
+        run_command("Yosys", "yosys_rewrite.log", command)
+    else:
+        ys_rewrite_params = {
+            "INPUT_BLIF": args.top_module+".blif",
```

Comment:
@ganeshgore : Is this really required? If you see https://github.com/lnis-uofu/OpenFPGA/blob/master/openfpga_flow/misc/qlf_yosys.ys, you will notice that I already added the below command to generate verilog:
write_verilog -noattr -nohex ${TOP_MODULE}.v

So, in the single yosys run, after blif is created, yosys will also generate verilog file. 

If you really want to change the verilog file name then you can make update to the above mentioned script.

Reply:
Also, to verify if this approach works, please update the task: https://github.com/lnis-uofu/OpenFPGA/tree/master/openfpga_flow/tasks/quicklogic_tests/lut_adder_test/config, to run simulation, which is currently commented.

Reply:
@lnsharma I agree. Now the rewrite script is not actually needed. We should think about deprecating it.

Reply:
@tangxifan I agree let me know if you would like to deprecate that part. 
But I guess you still need rewrite script during blif flow to convert ,blif design to verilog for functional simulation

Reply:
@ganeshgore Thanks for the reminder. I should keep the rewriting for ``vpr_blif`` flow and deprecate it for ``yosys_vpr_flow`` only